### PR TITLE
Added orgqr

### PR DESCRIPTION
**Pull request by tfalders**
Adds orgqr to hipSOLVER

### CODE REVIEW

**Review by jzuniga-amd — APPROVED** [review]
LGTM

File: library/src/hcc_detail/hipsolver.cpp [hunk 1]
```diff
@@ -173,6 +173,227 @@ catch(...)
...
+                                  int*              devInfo)
+try
+{
+    if(work != nullptr)
+        rocblas_set_workspace((rocblas_handle)handle, work, lwork);
+    else
+    {
+        if(!rocblas_is_managing_device_memory((rocblas_handle)handle))
+            rocblas_set_workspace((rocblas_handle)handle, nullptr, 0);
+    }
```
Comment (cgmb) [inline library/src/hcc_detail/hipsolver.cpp:295]: It's rather trivial, but would `else if` make sense here?

**Review by cgmb — APPROVED** [review]
Minor comment / question. Does not need to be addressed.

File: library/src/hcc_detail/hipsolver.cpp (hunk 1 above)
Comment (tfalders) [reply library/src/hcc_detail/hipsolver.cpp:295]: That would look nicer, I think. It'll affect all the other functions too, so I'll probably wait until they're all in and open a new PR. Should be able to do a search-and-replace.

**Pull request merged**